### PR TITLE
Neoterm update

### PR DESCRIPTION
**Pull request by stoeffel**

### CODE REVIEW

File: brangelina.vim [hunk 1]
```diff
@@ -72,6 +72,8 @@ let g:ale_elm_make_use_global=1
 let g:deoplete#enable_at_startup = 1
 let g:asyncrun_open = 8
 let g:neoterm_autoscroll = 1
+let g:neoterm_autoinsert = 1
+let g:neoterm_default_mod = 'tab'
```
Comment (stoeffel) [inline brangelina.vim:76]: @jwoudenberg how do you like this?

Comment (jwoudenberg) [reply brangelina.vim:76]: I still prefer full screen, failing that a split screen.

Sounds like something we could easily overwrite in our init files though, so also okay with this.

File: brangelina.vim [hunk 2]
```diff
@@ -138,10 +135,9 @@ nnoremap <leader>gg :Twiggy<cr>
 augroup customCommands
   autocmd!
   autocmd FileType markdown nnoremap <localleader>m :LivedownToggle<cr>
-  autocmd FileType javascript nnoremap <localleader>c :JSContextColorToggle<cr>
+  autocmd FileType javascript nnoremap <localleader>co :JSContextColorToggle<cr>
```
Comment (jwoudenberg) [inline brangelina.vim:138]: Also happy to remove this plugin. I'm not using it anymore.

Comment (stoeffel) [reply brangelina.vim:138]: I think there is more stuff that we could remove, wanna go over it some time?

Comment (jwoudenberg) [reply brangelina.vim:138]: Sounds good!

**Comment by stoeffel** [comment]
:tada: I found this gem https://gist.github.com/bfredl/e5b05193304a3340e29e998662ffff76#file-nvim-terminal-edit-py.
I've modified it a bit so that I can use that as my `$EDITOR`. https://github.com/stoeffel/.dots/blob/master/nvim-terminal-edit.py
https://github.com/stoeffel/.dots/blob/master/config.fish#L10 and for git https://github.com/stoeffel/.dots/blob/master/gitconfig#L26
@jwoudenberg

**Comment by stoeffel** [comment]
@jwoudenberg okay if we merge this?

**Comment by jwoudenberg** [comment]
Oh, yes, sorry! Merge ahead!

**Pull request merged**